src/DynamicMvvm/Command/DynamicCommandBuilder.cs

```diff
@@ -17,20 +17,25 @@ public class DynamicCommandBuilder : IDynamicCommandBuilder
 		/// </summary>
 		/// <param name="name">The name of the command.</param>
 		/// <param name="baseStrategy">The base strategy to use for the command.</param>
-		public DynamicCommandBuilder(string name, IDynamicCommandStrategy baseStrategy)
+		/// <param name="viewModel">The <see cref="IViewModel"/> that will own the newly created <see cref="IDynamicCommand"/>.</param>
+		public DynamicCommandBuilder(string name, IDynamicCommandStrategy baseStrategy, IViewModel viewModel = null)
 		{
 			if (string.IsNullOrEmpty(name))
 			{
 				throw new ArgumentException($"'{nameof(name)}' cannot be null or empty", nameof(name));
 			}
 
 			Name = name;
+			ViewModel = viewModel;
```

Comment:
No issue if viewModel is actually null here for future usage ?

Reply:
No issue if null. I added more xml doc (in the interface) about it.